2024-02-05 12:18:30.515 [info] Started local extension host with pid 5236.
2024-02-05 12:18:30.590 [error] Extension 'github.copilot-chat' wants API proposal 'interactiveUserActions' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned.
2024-02-05 12:18:30.591 [error] Extension 'github.copilot-chat' wants API proposal 'readonlyMessage' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned.
2024-02-05 12:18:30.591 [error] Extension 'github.copilot-chat' wants API proposal 'chatVariables' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned.
2024-02-05 12:18:30.591 [error] Extension 'ms-python.python' wants API proposal 'saveEditor' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned.
2024-02-05 12:18:33.025 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:18:33.314 [info] [perf] Render performance baseline is 24ms
2024-02-05 12:18:34.195 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:19:48.081 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:15.125 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:16.270 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:22.834 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:23.990 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:31.796 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:33.971 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:37.461 [error] Unable to read file '/home/elisena/Code/SkripsiL/2simulasi/Kalibrasi/kalibrasiSimpel/1kalibrasiSimpel.py' (Error: File not modified since): Error: Unable to read file '/home/elisena/Code/SkripsiL/2simulasi/Kalibrasi/kalibrasiSimpel/1kalibrasiSimpel.py' (Error: File not modified since)
    at n.F (vscode-file://vscode-app/opt/visual-studio-code/resources/app/out/vs/workbench/workbench.desktop.main.js:657:19875)
    at n.D (vscode-file://vscode-app/opt/visual-studio-code/resources/app/out/vs/workbench/workbench.desktop.main.js:657:19763)
    at async f.M (vscode-file://vscode-app/opt/visual-studio-code/resources/app/out/vs/workbench/workbench.desktop.main.js:2203:42897)
    at async f.readStream (vscode-file://vscode-app/opt/visual-studio-code/resources/app/out/vs/workbench/workbench.desktop.main.js:2203:42654)
    at async L.Cb (vscode-file://vscode-app/opt/visual-studio-code/resources/app/out/vs/workbench/workbench.desktop.main.js:2192:13459)
    at async L.resolve (vscode-file://vscode-app/opt/visual-studio-code/resources/app/out/vs/workbench/workbench.desktop.main.js:2192:11906)
    at async vscode-file://vscode-app/opt/visual-studio-code/resources/app/out/vs/workbench/workbench.desktop.main.js:2203:38766
2024-02-05 12:21:22.481 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:21:23.641 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:21:42.614 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:16:22.928 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:17:47.444 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:18:11.077 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:18:13.372 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:18:18.288 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:18:20.122 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:19:19.594 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:22.831 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:24.017 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:25.860 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:26.911 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:28.516 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:38.115 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:39.496 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:41.962 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:45.147 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:47.828 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:49.578 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 14:35:08.195 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 14:36:21.884 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
